jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -86,6 +86,7 @@ void jbmc_parse_optionst::set_default_options(optionst &options)
   options.set_option("assertions", true);
   options.set_option("assumptions", true);
   options.set_option("built-in-assertions", true);
+  options.set_option("lazy-methods", true);
```

Comment:
I guess we should get #2907 merged first...

Reply:
Has been merged